lib/Cake/Model/Model.php

```diff
@@ -2895,6 +2895,10 @@ public function exists($id = null) {
 			return false;
 		}
 
+		if ($this->useTable === false) {
+			return true;
```

Comment:
My thinking was, try to maintain the `'on' => 'create'` & `'on' => 'update'` in the validator. So if there is no id set then it returns false like usual. When the id is set, I was considering it a 'simulation' of an update call, therefore return true.

It's just suggestion, I can change it to return false if you think it's an over complication?